### ISSUE TITLE
Honoring exclude-from-graph-view

### ISSUE THREAD

**Issue by minxu-nyc** [sosuisen/logseq-cardbox#6]
Thanks for this nice plugin. Is it possible to exclude "exclude-from-graph-view:: true" pages in the card box?

**Comment by sosuisen** [comment]
That opinion seems reasonable. I will consider it in a near update.

**Comment by YU000jp** [comment]
There is one way not to use the property.
In the plugin settings, the user enters page names in the textarea, separated by new lines. It can also be added from the page menu.
Get it as an array and exclude it.

**Comment by sosuisen** [comment]
That's right. The plugin setting is also a good choice. 

What does "the page menu" mean? Does that mean it can be added directly from the card box panel?

**Comment by YU000jp** [comment]
Right-click on the title on each page to bring up a context menu. Add an item there. Once the item is selected, it will be updated in the settings.
https://plugins-doc.logseq.com/logseq/App/registerPageMenuItem

**Comment by sosuisen** [comment]
Thank you! I didn't know I could right-click here!
I also figured out how to access it from the plugin. This is very helpful.
